examples/fundamentals__dynamic-tests-from-api/cypress/integration/users-spec.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+// we can access the Cypress.env() object outside the test
+const users = Cypress.env('users')
+
+describe('Users from API', () => {
+  before(() => {
+    // confirm the users data
+    expect(users).to.be.an('array').and.to.have.length(3)
+
+    cy.visit('index.html')
```

Comment:
I would recommend moving this `cy.visit()` into a `beforeEach` since this is not recommended, especially going forward.